source/FreeRTOS_UDP_IP.c

```diff
@@ -282,12 +282,12 @@ BaseType_t xProcessReceivedUDPPacket( NetworkBufferDescriptor_t * pxNetworkBuffe
     if( pxUDPPacket->xEthernetHeader.usFrameType == ipIPv4_FRAME_TYPE )
     {
         xReturn = xProcessReceivedUDPPacket_IPv4( pxNetworkBuffer,
-                                                  usPort, pxIsWaitingForARPResolution );
```

Comment:
Is this from uncrustify?

Reply:
no, that line had a conflict with HEAD of dev/v6 branch, the conflict resolution added that change from either upstream or local